File: cpp/Aksesoris.cpp
#include <iostream>
#include <string>
#include "PetShop.cpp"

using namespace std;

//deklarasi class yang menjadi anak dari Petshop
class Aksesoris : public PetShop 
{
    //atribut private
    private:
        string jenis;
        string bahan;
        string warna;

    public:
    //konstruktor
    Aksesoris()
    {
        this->jenis = "";
        this->bahan = "";
        this->warna = "";
    }

    Aksesoris(string jenis, string bahan, string warna)
    {
        this->jenis = jenis;
        this->bahan = bahan;
        this->warna = warna;
    }

    //setter
    void setJenis(string jenis)
    {
        this->jenis = jenis;
    }
    void setBahan(string bahan)
    {
        this->bahan = bahan;
    }
    void setWarna(string warna)
    {
        this->warna = warna;
    }

    //getter
    string getJenis()
    {
        return this->jenis;
    }
    string getBahan()
    {
        return this->bahan;
    }
    string getWarna()
    {
        return this->warna;
    }

    //destructor
    ~Aksesoris()
    {
        
    }
};

File: python/Aksesoris.py
#library
from PetShop import PetShop

#deklarasi class yang menjadi anak dari PetShop
class Aksesoris(PetShop):
    #atribut private
    __jenis = ""
    __bahan = ""
    __warna = ""

    #constructor
    def __init__(self, jenis = "", bahan = "", warna = ""):
        self.jenis = jenis
        self.bahan = bahan
        self.warna = warna

    #setter
    def setjenis(self, jenis):
        self.__jenis = jenis
    def setbahan(self, bahan):
        self.__bahan = bahan
    def setwarna(self, warna):
        self.__warna = warna

    #getter
    def getjenis(self):
        return self.__jenis
    def getbahan(self):
        return self.__bahan
    def getwarna(self):
        return self.__warna